README.md

```diff
@@ -14,30 +14,76 @@ This README contains a high level description of contributing a SWIP, for conven
-a discussion forum or open GitHub issue where people can discuss the SWIP as a whole.
+A SWIP proposal or suggested edits (or updates) may come from anyone (inside or outside the organisation) and are received in the form of a pull request to the [SWIP repository](https://github.com/ethersphere/SWIPs). Throughout the lifetime of a SWIP from proposal to final accepted, we use the PR-review infrastructure of github to comment and improve the SWIP.
 
-If your SWIP requires images, the image files should be included in a subdirectory of the `assets` folder for that SWIP as 
-follows: `assets/swip-X` (for swip **X**). When linking to an image in the SWIP, use relative links such as 
-`../assets/swip-X/image.png`.
+THe github permissions of the repo reflect these privileges:
+- Anyone can open a new PR.
+- Only SWIP editors can merge a PR to master.
 
-When you believe your SWIP is mature and ready to progress past the draft phase, you should open a PR changing the state of your 
-SWIP to 'Final'. An editor will review your draft and ask if anyone objects to its being finalised. If the editor decides there 
-is no rough consensus - for instance, because contributors point out significant issues with the SWIP - they may close the PR and 
-request that you fix the issues in the draft before trying again.
+### Creating a new SWIP
 
-For a detailed description of the SWIP process see [SWIP-0](SWIPs/swip-0.md).
+A new SWIP can be proposed by opening a PR in the [SWIP repo](https://github.com/ethersphere/SWIPs) titled `SWIP-<XXXX>: <title>` where `XXXX` is the SWIP number left padded with zeros up to four digits (i.e., fprinted with the format string `%04d`). 
+The PR is adding at least a new placeholder file for the new proposal named `XXXX.md` to be located in the `SWIPs` subdirectory of the SWIPs repo.
 
-# SWIP Status Terms
+This placeholder file contains minimally only the the yaml header with the following fields:
 
-* **Draft** - a SWIP that is undergoing rapid iteration and changes.
-* **Last Call** - a SWIP that is done with its initial iteration and ready for review by a wide audience.
-* **Accepted** - a core SWIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author. The process for Core Devs to decide whether to encode a SWIP into their clients as part of a mandatory update is not part of the SWIP process. If such a decision is made, the SWIP will move to final.
-* **Final (non-Core)** - a SWIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author.
-* **Final (Core)** - a SWIP that the Core Devs have decided to implement and release in a future mandatory update or has already been released as such. 
-* **Deferred** - a SWIP that is not being considered for immediate adoption. May be reconsidered in the future.
+```yaml
+ ---
+title: fairx
```

Comment:
This example title is concise, but not descriptive. I suggest taking one of the existing as example:
Neighbourhood hopping